colors/onedark.vim

```diff
@@ -596,6 +596,7 @@ call s:h("CocErrorSign", { "fg": s:red })
 call s:h("CocWarningSign", { "fg": s:yellow })
 call s:h("CocInfoSign", { "fg": s:blue })
 call s:h("CocHintSign", { "fg": s:cyan })
+call s:h("CocFadeOut", { "fg": s:special_grey })
```

Comment:
```suggestion
call s:h("CocFadeOut", { "fg": s:comment_grey })
```

I fear `special_grey` may be too hard to read for this usage. What do you think of `comment_grey`?

Reply:
Sure, seems reasonable to me.